charts/victoria-metrics-k8s-stack/templates/rules/vm-health.yaml

```diff
@@ -58,8 +64,8 @@ spec:
     - alert: TooHighMemoryUsage
       annotations:
         description: Too high memory usage may result into multiple issues such as OOMs or degraded performance. Consider to either increase available memory or decrease the load on the process.
-        summary: It is more than 90% of memory used by "{{`{{`}} $labels.job {{`}}`}}"("{{`{{`}} $labels.instance {{`}}`}}") during the last 5m
-      expr: (process_resident_memory_anon_bytes / vm_available_memory_bytes) > 0.9
+        summary: It is more than 80% of memory used by "{{`{{`}} $labels.job {{`}}`}}"("{{`{{`}} $labels.instance {{`}}`}}") during the last 5m
```

Comment:
Is the original 90% on purpose? Below [TooHighCPUUsage](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-k8s-stack/templates/rules/vm-health.yaml#L69) also use it as threshold. @f41gh7 @k1rk

Reply:
The updated rules files were generated by running the sync_rules.py after my changes. This change was pulled from https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/alerts-health.yml